To: vim_dev@googlegroups.com
Subject: Patch 7.3.334
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.334
Problem:    Latest MingW about XSUBPP referencing itself. (Gongqian Li)
Solution:   Rename the first use to XSUBPPTRY.
Files:	    src/Make_ming.mak


*** ../vim-7.3.333/src/Make_ming.mak	2011-09-30 16:56:00.000000000 +0200
--- src/Make_ming.mak	2011-10-09 14:23:06.000000000 +0200
***************
*** 108,117 ****
  # on NT, it's here:
  PERLLIB=$(PERL)/lib
  PERLLIBS=$(PERLLIB)/Core
! XSUBPP=$(PERLLIB)/ExtUtils/xsubpp
! XSUBPP_EXISTS=$(shell perl -e "print 1 unless -e '$(XSUBPP)'")
  ifeq "$(XSUBPP_EXISTS)" ""
! XSUBPP=perl $(XSUBPP)
  else
  XSUBPP=xsubpp
  endif
--- 108,117 ----
  # on NT, it's here:
  PERLLIB=$(PERL)/lib
  PERLLIBS=$(PERLLIB)/Core
! XSUBPPTRY=$(PERLLIB)/ExtUtils/xsubpp
! XSUBPP_EXISTS=$(shell perl -e "print 1 unless -e '$(XSUBPPTRY)'")
  ifeq "$(XSUBPP_EXISTS)" ""
! XSUBPP=perl $(XSUBPPTRY)
  else
  XSUBPP=xsubpp
  endif
*** ../vim-7.3.333/src/version.c	2011-10-04 21:22:40.000000000 +0200
--- src/version.c	2011-10-12 14:10:20.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     334,
  /**/

-- 
Q: Should I clean my house or work on Vim?
A: Whatever contains more bugs.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
